src/styles/common.scss

```diff
@@ -280,6 +280,20 @@
     text-align: center;
 }
 
+@media (max-width: $desktop-width) {
+    .linkCardTitle {
+        font-size: 30px;
+    }
+}
+
+@media (max-width: $tablet-width) {
+    .linkCardTitle {
+        font-family: var(--text-font);
+        font-size: 22px;
+        font-weight: 600;
+    }
+}
```

Comment:
On mobile the cards are larger than on tablet. I don't see the point of switching to a smaller font for larger cards.